tasks/time-series/time-series-forecasting/train_weathernet.py

```diff
@@ -0,0 +1,168 @@
+# Imports
+import numpy as np
+import tensorflow as tf
+from sklearn.preprocessing import MinMaxScaler
+import keras
+from keras.models import Model
+from keras.layers import Dense, CuDNNLSTM, Input
+import datetime
+import os
+import shutil
+from numpy.random import seed
+from tensorflow import set_random_seed
+import matplotlib.pyplot as plt
+import argparse
+import mlflow.sklearn
+from mlflow import log_metric
+import mlflow.keras
+import warnings
+import sys
+
+# Initialize
+# Keras and Tensorflow Random Numbers
+seed(1)
+set_random_seed(2)
+
+# Clear
+# Previous Tensorflow Session
+tf.keras.backend.clear_session()
+
+arr = sys.argv
+# Define Variables
+lookback_window = 32
+steps_per_epoch = 364 * 10
+epochs = 30
+val_steps = 364 * 2
+LSTMunits = 100
+predict_steps = 364 * 4
+predict_shift = 0
+city = sys.argv[1]
+print ('Running Weathernet for '+city)
+# Working directories
+model_filepath = 'Weather_Net_'+city+'.hf5'
+image_dir = 'images/'
```

Comment:
Create permanent sub directory, if intended otherwise it's throwing error for image creation.

Reply:
Done.

Reply:
May I please request of not versioning image files within directory. You could remove them or additionally update [.gitignore](https://github.com/Doctor-Fu/ai-platform/blob/master/.gitignore) so next time it will ignore automatically.

Reply:
Commited update .gitignore.